docs/.vitepress/config.mjs

```diff
@@ -0,0 +1,204 @@
+              //newEditLink defines a frontmatter key you can use to append a path to main
+              return `https://github.com/mavlink/MAVSDK-docs/edit/main/${frontmatter.newEditLink}`;
+            } else {
+              return `https://github.com/mavlink/MAVSDK-docs/edit/main/${filePath}`;
+            }
+          },
+          text: "Edit on GitHub",
+        },
+      },
+    },
+
+    /*
+    zh: {
+      label: "中文 (Chinese)",
+      lang: "zh-CN", // optional, will be added  as `lang` attribute on `html` tag
+      themeConfig: {
+        sidebar: getSidebar.sidebar("zh"),
+      },
+      // other locale specific properties...
+    },
+    ko: {
+      label: "한국어 (Korean)",
+      lang: "ko-KR", // optional, will be added  as `lang` attribute on `html` tag
+      themeConfig: {
+        sidebar: getSidebar.sidebar("ko"),
+      },
+
+      // other locale specific properties...
+    },
+    */
+  },
+
+  //
+
+  themeConfig: {
+    // https://vitepress.dev/reference/default-theme-config
+    siteTitle: false,
+    logo: {
+      light: "/site/logo_mavsdk_header_lightmode.png",
+      dark: "/site/logo_mavsdk_header_darkmode.png",
+      alt: "",
+    },
+    sidebar: getSidebar.sidebar("en"),
+    externalLinkIcon: true,
+
+    editLink: {
+      pattern: "https://crowdin.com/project/mavsdk",
+      text: "Edit translation on Crowdin",
+    },
+
+    search: {
+      provider: "local",
+    },
+
+    nav: [
+      {
+        text: "MAVSDK",
+        items: [
+          {
+            text: "Website",
+            link: "https://dronecode.org/sdk/",
+            ariaLabel: "MAVSDK website link",
+          },
+          {
+            text: "Source Code",
+            link: "https://github.com/mavlink/MAVSDK",
+          },
+        ],
+      },
+      {
+        text: "Dronecode",
+        items: [
+          {
+            text: "PX4",
+            link: "https://px4.io/",
+          },
+          {
+            text: "QGroundControl",
+            link: "http://qgroundcontrol.com/",
+          },
+          {
+            text: "MAVLINK",
+            link: "https://mavlink.io/en/",
+          },
+          {
+            text: "PX4 Guide",
+            link: "https://docs.px4.io/main/en/index.html",
+          },
+          {
+            text: "QGroundControl Guide",
+            link: "https://docs.qgroundcontrol.com/master/en/qgc-user-guide/",
+          },
+          {
+            text: "Dronecode Camera Manager",
+            link: "https://camera-manager.dronecode.org/en/",
+          },
+        ],
+      },
+      {
+        text: "Support",
+        link: "https://discuss.px4.io/c/mavsdk/19",
+      },
+      {
+        text: "Version",
+        items: [
+          {
+            text: "main",
+            link: "https://mavsdk.mavlink.io/main/en/cpp/api_changes.html",
+          },
+          {
+            text: "v2",
+            link: "https://mavsdk.mavlink.io/v2.0/en/cpp/api_changes.html",
+          },
```

Comment:
I will create v3 and we should add it here, because main won't be v3 forever.

Reply:
We should create main, then branch it to create v3. At that point we'd create this link

Reply:
Though first we'd fix up main to have the right API reference - that being your job after this merges.

Reply:
Right, that's fine.